src/compound/SupplyVaultBase.sol

```diff
@@ -83,12 +99,17 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
 
     /// EXTERNAL ///
 
-    function transferTokens(
-        address _asset,
-        address _to,
-        uint256 _amount
-    ) external onlyOwner {
-        ERC20(_asset).safeTransfer(_to, _amount);
+    function setRecipient(address _recipient) external onlyOwner() {
+        if (_recipient == address(0)) revert ZeroAddress();
```

Comment:
Perhaps we'll want to disable the rewards transfer behavior in the future. So we may not want to prevent setting the `recipient` to `address(0)`

Reply:
Yes but removing this does not prevent to transfer tokens, so I'm not sure this is a solution..

Reply:
I would say that it is possible to set the recipient to 0, but not to transfer to recipient if it is 0 (this part has also been implemented). 

Plus, it prevents transferring rewards if you forget to set the recipient.